### DISCUSSION TITLE
Notification opens screen with incorrect data

### DISCUSSION THREAD

**Discussion by revolter** [community/community#103176] [Mobile]
### Select Topic Area

Bug

### Body

I received a notification which opened:

![IMG_8758](https://github.com/community/community/assets/5748627/6fcc21eb-14ee-42cf-99a1-8f9c87e346d0)

but because "Paste From History" is not the name of my plugin, I used the share button in the top right corner, and in the browser it displayed the correct data:

![IMG_8759](https://github.com/community/community/assets/5748627/fb18ded7-2045-476a-932c-35a436b7b1cf)

<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.147.0 (121573070)
Device: iPhone15,3 (iOS 17.3)
</details>

**Comment by Ai1972** [comment]
Mobile

**Comment by Ai1972** [comment]
Bug
